### DISCUSSION TITLE
Where do configurations go?

### DISCUSSION THREAD

**Discussion by donahchoo** [spring-projects/spring-modulith#607] [Q&A]
Hi,

In a modulith application, where should configurations go?  I have configurations for jetty, spring security, etc that don't really belong in a module.

thanks!

**Comment by breun** [comment]
I think there are several options. The easiest would be to put them inside or next to your application class in the same package.

If you want to put configuration classes in a different package, you'll don't want Spring Modulith to consider that package a Spring Modulith module. You can achieve that in several ways again. You could tell Spring Modulith to [only consider explicitly annotated modules or implementing your own module detection strategy](https://docs.spring.io/spring-modulith/reference/fundamentals.html#customizing-modules), or I guess you could make it an [open module](https://docs.spring.io/spring-modulith/reference/fundamentals.html#modules.advanced.open). Or you could put the configurations in a package that is not under your application class and explicitly import them from that package.
